code-generation/llms-vscode-integration.md

```diff
@@ -0,0 +1,87 @@
+
+Before configuring the "Continue" extension, ensure that the Model Service is up and running. Follow the instructions provided in the existing [README.md](README.md) document to build and deploy the Model Service. Note the port and endpoint details for the Model Service.
+
+## Step 3: Configure "Continue" config.json File
+
+Once the Model Service is operational, configure the "Continue" extension. Open VS Code and access the Command Palette by pressing `Ctrl+Shift+P` (Windows/Linux) or `Cmd+Shift+P` (Mac). Type "Continue: Configure" and select the option to open the configuration file. Edit the `config.json` file with the appropriate configuration.
+
+```
+{
+  "title": "YourTitleHere",
+  "model": "YourModelName",
+  "completionOptions": {},
+  "apiBase": "http://localhost:8001/v1/",
+  "provider": "openai"
+}
+```
+
+# Interacting with the "Continue" Extension: Practical Examples
+
+Now that you've configured the "Continue" extension, let's explore how you can effectively interact with the language model directly within VS Code. Here are several ways to engage with the extension:
+
+1. **Prompting for code generation:** Open the "Continue" panel in VS Code and prompt the extension with a specific task, such as "Write a code to add two numbers." The extension will then provide relevant code autocompletion suggestions based on your input prompt, aiding in code generation and text completion tasks.
+
+   ![Prompt-response](../assets/interaction-vscode1.png)
+
+2. **Querying Working Code:** Copy your existing code snippet or press `⌘ + L` to paste it into the "Continue" panel, then pose a question such as "Explain this section of the code." The extension (LLM) will analyze the code snippet and provide explanations or insights to help you understand it better.
+
+   ![Querying Working Code](../assets/interaction-vscode2.png)
+
+3. **Editing Code in Script:** Edit your Python code directly within a `.py` script file using the "Continue" extension. Press `⌘ + I` to initiate the edit mode. You can then refine a specific line of code or request enhancements to make it more efficient. The extension will suggest additional code by replacing your edited code and provide options for you to accept or reject the proposed changes.
+
+   ![Editing Code in Script](../assets/interaction-vscode3.png)
+
+By exploring these interactions, users can fully leverage the capabilities of language models within VS Code, enhancing their coding experience and productivity.
+
+4. **Tab Autocomplete:**
+
+![autocompletion-config-example](../assets/autocomplete_example.png)
+
+In addition to its core functionalities, the "Continue" extension offers a tab auto complete feature in its pre-release version. This feature enhances the coding experience by providing aut-complete suggestions tailored to your coding context within VS Code. To leverage this functionality with the custom model, follow these steps to configure the `config.json` file:
```

Comment:
I'm not sure we need this section here. This is mainly just generic info you can get from the the Continue docs, right? 

(Its good info to know, but don't think its that relevant here)

Reply:
Thanks for the feedback. It does make sense. Removed it and updated the file.
Please take a look. Thank you @MichaelClifford